builtin/v13/market/market_state.go

```diff
@@ -55,6 +56,14 @@ type State struct {
 
 	// Verified registry allocation IDs for deals that are not yet activated.
 	PendingDealAllocationIds cid.Cid // HAMT[DealID]AllocationID
+
+	/// Maps providers to their sector IDs to deal IDs.
+	/// This supports finding affected deals when a sector is terminated early
+	/// or has data replaced.
+	/// Grouping by provider limits the cost of operations in the expected use case
+	/// of multiple sectors all belonging to the same provider.
+	/// HAMT[ActorID]HAMT[SectorNumber]SectorDealIDs
```

Comment:
```suggestion
	/// HAMT[ActorID]HAMT[SectorNumber][]DealID
```

the FIP now has it described in this form

Reply:
actually I guess the named type makes more sense here so maybe it's reasonable to leave it like this